cockatrice/src/player.cpp

```diff
@@ -2685,6 +2686,28 @@ void Player::updateCardMenu(const CardItem *card)
             cardMenu->addAction(aClone);
         }
     }
+    addRelatedCardView(card, cardMenu);
```

Comment:
While this location is good for cards in hand, it's really inconvenient for cards in play. Either we should make this feature only work in hand, or we should place it somewhere better when the card's in play. Similar issues on the stack.

<img width="327" alt="screenshot 2018-02-16 02 35 21" src="https://user-images.githubusercontent.com/7460172/36297685-27ae95b2-12c2-11e8-82e0-0cddcb2e3988.png">
<img width="358" alt="screenshot 2018-02-16 02 35 25" src="https://user-images.githubusercontent.com/7460172/36297686-27c95906-12c2-11e8-9a7b-4cc23d5afa58.png">
<img width="346" alt="screenshot 2018-02-16 02 35 51" src="https://user-images.githubusercontent.com/7460172/36297687-27d640da-12c2-11e8-89bc-3b9c498e56af.png">

Reply:
For cards in play I'd actually put it near the token menu, maybe above or below

Reply:
My thought was to `view related card images >` submenu it